ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg

```diff
@@ -0,0 +1,4297 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
missing license/origin in the SVG

Reply:
also it's strange it seems there is PNG in that SVG

Reply:
argh on Apache Camel irc chat, someone pointed me to this "false" svg, just an embedded png :'(

Reply:
I modified for the png, it is now a real svg.

I added the license but i still have build complaining about missing header:
[INFO] --- license-maven-plugin:3.0:check (check-headers) @ che-core-ide-stacks ---
[INFO] Checking licenses...
[WARNING] Unknown file extension: /home/codenvy/workspace/che-pullrequests-test-ocp/ide/che-core-ide-stacks/src/main/resources/stacks.json
[WARNING] Missing header in: /home/codenvy/workspace/che-pullrequests-test-ocp/ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg

Reply:
yes plugin is checking for Red Hat / EPL license

you need to exclude your file from the plug-in
https://github.com/eclipse/che/blob/ad6376a43644179ea04b0156b732e468855d5d9c/ide/che-core-ide-stacks/pom.xml#L33-L35

Reply:
fixed, thanks for the help